automerge-js/package.json

```diff
@@ -4,10 +4,21 @@
   "main": "src/index.js",
   "license": "MIT",
   "scripts": {
-    "test": "mocha --bail --full-trace"
+    "lint": "tslint --project tsconfig.json",
```

Comment:
Due to [deprecation of tslint in 2019](https://blog.palantir.com/tslint-in-2019-1a144c2317a9), any reason not to prefer [`typescript-eslint`](https://typescript-eslint.io)? I can help with any setup issues that might come up, we've recently done a migration.

Reply:
Thank you!  I haven't done much typescript work so I'll take all the help i can get.  Since I did a pretty fast and rough javascript to typescript conversion were a long way from the lint passing but I really wanted to have it in here as a bar to reach.

Reply:
Hey Scott!  I mean... this PR is going to get merged with probably 200+ lint errors in the TODO list... A PR that fixed any one of those would be warmly welcomes 😂